ansible_ai_connect/ai/api/views.py

```diff
@@ -176,9 +176,18 @@
 
 
 class AACSAPIView(APIView):
+    def __init__(self):
```

Comment:
This just removes some warnings about setting properties later, outside the `__init__` function.